src/main/java/io/strimzi/ValidatingWebhook.java

```diff
@@ -65,6 +63,10 @@ private ObjectMeta extractEvictionMetadata(AdmissionRequest request)    {
         }
     }
 
+    private boolean matchingLabel(Map<String, String> labels) {
+        return labels.containsKey(KIND_LABEL_KEY) && labels.get(KIND_LABEL_KEY).equals(KIND_LABEL_VALUE);
```

Comment:
Can `labels` be null`? If not, you can simply do this:
```
return KIND_LABEL_VALUE.equals(labels.get(KIND_LABEL_KEY))
```

If it can be null, you should do something like:
```
return labels != null && KIND_LABEL_VALUE.equals(labels.get(KIND_LABEL_KEY))
```